templates/answerfile_4.2_upgrade.txt.j2

```diff
@@ -0,0 +1,11 @@
+{% include "./templates/basic_answerfile.txt.j2" %},
+OVESETUP_OVN/ovirtProviderOvn=bool:True
```

Comment:
This should be provided as role parameter

Reply:
Done